packages/vite/src/utils/generator-utils.ts

```diff
@@ -756,11 +667,10 @@ function handleViteConfigFileExists(
       formats: ['es', 'cjs'],
     },
     rollupOptions: {
-      external: [
```

Comment:
Here was the second errors.

The possible values were `["'react', 'react-dom', 'react/jsx-runtime'"] ` or `[]`.

They are now ['react', 'react-dom', 'react/jsx-runtime'] or `[]`.